tests/README.md

```diff
@@ -20,9 +20,39 @@ This script creates `versionsTest` directory to which zstd repository is cloned.
 Then all tagged (released) versions of zstd are compiled.
 In the following step interoperability between zstd versions is checked.
 
+#### `automated-benchmarking.py` - script for benchmarking zstd prs to dev
+
+This script benchmarks facebook:dev and changes from pull requests made to zstd and compares
+them against facebook:dev to detect regressions. This script currently runs on a dedicated
+desktop machine for every pull request that is made to the zstd repo but can also
+be run on any machine via the command line interface.
+
+There are three modes of usage for this script: mode 0 will fetch all open pull requests
+in the repo and benchmark them against dev, mode 1 will fetch the pull request
+associated with the current hash and benchmark that against dev, and mode 3 will
```

Comment:
mode 0, mode 1, mode 3, 
but no mode 2 ?

Reply:
This will be clearer with labels